README.md

```diff
@@ -99,68 +117,72 @@ Biometry Exception Types:
 
 If it's a cancellation error, `OperationCanceledException` is thrown.
 
-### GetGapabilites
+#### GetGapabilites
 
 Gets the device's current biometric capabilities.
 
 It will return a struct `BiometryCapabilities` with the detailled device configuration.
 
-### ScanBiometry
+#### ScanBiometry
 
 Attemps to scan the user's biometry.
 
 ``` cs
 await biometryService.ScanBiometry(cancellationToken);
```

Comment:
I think we should use CancellationToken.None for the code samples so that in can be copy and pasted more easily.